src/framework/preferences/css/stylus/utils/Fonts.styl

```diff
@@ -0,0 +1,81 @@
+// Contains utility functions for changing font family
+// !important is used for all styles to ensure that Preferences Framework's enactors are able to override the pages default styles.
+
+fonts = {
+    ".fl-font-arial": {
+        fontFamily: "Arial"
+    }
+    ".fl-font-verdana": {
+        fontFamily: "Verdana"
+    }
+    ".fl-font-times": {
+        fontFamily: "Georgia, Times, Times New Roman, serif"
+    }
+    ".fl-font-comic-sans": {
+        fontFamily: "Comic Sans MS, sans-serif"
+    }
+}
+
+build-fonts-prefsEditor(textFont) {
+
+	for fontName in textFont {
+		font = textFont[fontName].fontFamily;
+
+		.fl-prefsEditor {
+			{fontName},
+			{fontName} * {
+				:not([class*='icon']) {
+					font-family: unquote(font) !important;
+                }
+			}
+		}
+	}
+}
+
+build-fonts-separatedPanelPrefsEditorFrame(textFont) {
+
+    for fontName in textFont {
+        font = textFont[fontName].fontFamily;
+
+        .fl-prefsEditor {
+            {fontName},
+            {fontName} * {
+                :not([class*='icon']) {
+                    font-family: unquote(font) !important;
+                }
+            }
+        }
+    }
+}
+
+
+build-fonts-separatedPanelPrefsEditor(textFont) {
+
+    for fontName in textFont {
+        font = textFont[fontName].fontFamily;
+
+        .fl-prefsEditor {
+            {fontName},
+            {fontName} * {
+                :not([class*='icon']) {
+                    font-family: unquote(font) !important;
+                }
+            }
+        }
+    }
+}
+
+
+build-fonts-Enactors(textFont) {
```

Comment:
Just for the record, @jobara, I think we should be thoughtful when applying code style conventions imported from other programming languages. Stylus clearly has a very different syntax, and you'll notice that the definition of anonymous functions in Stylus is unambiguous (due to the special <code>@</code> symbol), unlike in JavaScript. If I remember correctly, this was the rationale for the original Crockford-ist use of the space in JS function definitions that are on the right side of a variable assignment.

Plus, well, I don't think spaces in function definitions are syntactically valid in Stylus, as @grrrero noted. :)